bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/config/VeluxThingConfiguration.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.velux.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.openhab.binding.velux.VeluxBindingConstants;
+import org.openhab.binding.velux.things.VeluxProductSerialNo;
+
+/**
+ * The {@link VeluxThingConfiguration} is a wrapper for
+ * configuration settings needed to access the <B>Velux</B> device.
+ * <p>
+ * It contains the factory default values as well.
+ * <p>
+ * There are three parts. Information for:
+ * <UL>
+ * <LI>{@link #sceneName Channel of type scene},</LI>
+ * <LI>{@link #serial Channels of type actuator, rollershutter and window},</LI>
+ * <LI>{@link #sceneLevels Channel of type virtualshutter}.</LI>
+ * </UL>
+ *
+ * @author Guenther Schreiner - Initial contribution.
+ * @author Andrew Fiddian-Green - adapted.
+ */
+@NonNullByDefault
+public class VeluxThingConfiguration extends Configuration {
+
+    /**
+     * {@link #sceneName} of type {@link String}, identifying a Velux scene by human-readable name.
+     * <P>
+     * <B>Configuration for the channel scene:</B>
+     * </P>
+     * <UL>
+     * <LI>{@link #sceneName} for identification of a set of settings, so called scene.</LI>
+     * </UL>
+     */
+    @SuppressWarnings("unused")
```

Comment:
This class contains the properties for the Things, used within the handler class by
`configuration = getConfigAs(VeluxThingConfiguration.class);`

To avoid warnings of the compiler, the` @SuppressWarnings("unused")` statement is used at the definition.

Reply:
But that would mean that some configuration parameters are unused? Or is it because you do not fully use the configuration originating from the `getConfigAs`